src/views/Notes/NoteCreatePage.test.js

```diff
@@ -19,22 +27,66 @@ const history = {
       entityType: 'user',
       referredRecordData: {}
     }
-  }
+  },
+  'createHref': () => {},
+  'push': () => {},
+  'replace': () => {},
+  'go': () => {},
+  'goBack': () => {},
+  'goForward': () => {},
+  'block': () => {},
+  'listen': () => {},
 };
 
 const location = history.location;
 
-const renderNoteCreateRoute = (props = {
-  history,
-  location
-}) => render(
-  <NoteCreateRoute {...props} />
+const renderNoteCreateRoute = (props) => render(
+  <MemoryRouter>
+    <NoteCreateRoute {...props} />
+  </MemoryRouter>
 );
 
-describe('Note Create Page', () => {
-  it('should render Note Create Page', async () => {
-    const { getByText } = renderNoteCreateRoute();
+describe('Note Create Route', () => {
+  describe('should render Note Create Page', () => {
+    it('should hold appropriate values for referredRecordData', () => {
```

Comment:
this test doesn't check with what props NoteCreatePage is invoked, it tests  `retrieveNoteReferredEntityDataFromLocationState`